src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.Windows.cs

```diff
@@ -16,6 +16,15 @@ namespace System.Net.Security
 {
     internal static class SslStreamPal
     {
+        private static readonly bool UseNewCryptoApi = Is1809OrNewer();
+        private static bool Is1809OrNewer()
```

Comment:
Is the separate function needed anymore?
```C#
private static readonly bool UseNewCryptoApi = 
    // On newer Windows version we use new API to get TLS1.3.
    // API is supported since Windows 10 1809 (17763)but there is no reason to use at the moment.
    Interop.NtDll.RtlGetVersionEx(out Interop.NtDll.RTL_OSVERSIONINFOEX osvi) == 0 &&
    osvi.dwMajorVersion >= 10 && osvi.dwBuildNumber >= 18836;
```

Reply:
updated.